vm/runtime/runtime.go

```diff
@@ -28,10 +36,8 @@ func Fetch(from, i any) any {
 	// Structs, maps, and slices can be access through a pointer or through
 	// a value, when they are accessed through a pointer we don't want to
 	// copy them to a value.
-	if kind == reflect.Ptr {
-		v = reflect.Indirect(v)
```

Comment:
Looks like we going to loose `reflect.Indirect` here.

Reply:
Here is ooriginal change: https://github.com/antonmedv/expr/commit/6f0c8555bad3d5ab09f35c1ef3c3047382514e3c

Reply:
We are still doing the "indirect" in the `deref` function as we are calling `Elem()` of the value is a pointer.
Which should be doing the same as the `Indirect` function, which is also doing `Elem()` behind the scene:
```go
func Indirect(v Value) Value {
	if v.Kind() != Pointer {
		return v
	}
	return v.Elem()
}
```

Reply:
I just ran the benchrmak, it seems we are in the same order of magnitude:
before: 
```
Benchmark_largeStructAccess-12           1000000               134.4 ns/op
```
after:
```
Benchmark_largeStructAccess-12           1000000               132.2 ns/op
```

Reply:
Cool! let me check it a bit more as runtime changes should be done very carefully.